demo/run-osm-demo.sh

```diff
@@ -89,7 +89,9 @@ if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
 fi
 
 if [ "$PUBLISH_IMAGES" = true ]; then
+    docker network disconnect kind kind-registry
```

Comment:
I haven't had to do this since #4554 was merged.
more context: https://github.com/openservicemesh/osm/pull/4554#pullrequestreview-891801791